app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :likes
+  has_many :comments
+
+  def recent_comments
+    Comment.limit(5).order(created_at: :desc)
+  end
+
+  def post_counter
+    user.increment!(:PostsCounter)
+  end
```

Comment:
## Optional
- [ ] I suggest that you use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - 1after_save` to call the `#post_counter` method every time a new post has been saved to the database, then make the `post_counter` method private